### ISSUE TITLE
purefb_subnet - gateway required

### ISSUE THREAD

**Issue by gc-nathanh** [Pure-Storage-Ansible/FlashBlade-Collection#156]
Running v1.6.0 of the bundle.    The documents state that only 'name' is required, but if I omit 'gateway' I receive the below error:

`fatal: [localhost]: FAILED! => {"changed": false, "msg": "state is present but all of the following are missing: gateway"}
`

running Ansible 2.11.4:
```
ansible [core 2.11.4]
  config file = None
  configured module search path = ['/Users/nathanh/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/nathanh/venvs/tools-20210909/lib/python3.9/site-packages/ansible
  ansible collection location = /Users/nathanh/.ansible/collections:/usr/share/ansible/collections
  executable location = /Users/nathanh/venvs/tools-20210909/bin/ansible
  python version = 3.9.7 (default, Sep  3 2021, 12:37:55) [Clang 12.0.5 (clang-1205.0.22.9)]
  jinja version = 3.0.1
  libyaml = True
```

**Comment by sdodsley** [comment]
This is a known issue that was resolved with #147 
You can grab the fix from that PR - the fix will be included in the next release of the Collections.

**Comment by gc-nathanh** [comment]
Ok, thanks for the swift response!
